src/main/java/model/ladder/Point.java

```diff
@@ -0,0 +1,34 @@
+package model.ladder;
+
+import java.util.Random;
+
+public class Point {
+
+    private static final Random random = new Random();
+    private static final State[] STATES = {State.LINE, State.SPACE};
+    private final State state;
+
+    public Point(State state) {
+        this.state = state;
+    }
+
+    enum State{
+        BAR("|"), LINE("-"), SPACE(" ");
+        private String display;
+        State(String display){
+            this.display = display;
+        }
+    }
+
+    public boolean isLine(){
+        return (state == State.LINE);
+    }
+
+    public String getDisplay(){
+        return state.display;
+    }
+
+    public static State generateStateByRandom(){
+        return STATES[random.nextInt(2)];
```

Comment:
```suggestion
        return STATES[random.nextInt(STATES.length)];
```

로 작성하면 더 보기 좋을 것 같습니다 ㅎㅎ